compiler/damlc/daml-stdlib-src/DA/Action.daml

```diff
@@ -59,6 +59,15 @@ foldl1A : Action m => (a -> a -> m a) -> [a] -> m a
 foldl1A f (x :: xs) = foldlA f x xs
 foldl1A _ [] = error "foldl1M: empty list"
 
+-- | Filters the list using the applicative function: keeps only the elements where the predicate holds.
+--
+-- ```
+-- >>> filterA (\x -> Some (x % 2 == 0)) [0..5]
```

Comment:
This example seems more confusing than helpful. If you always return `Some` there is no point in using `filterA`.

Reply:
I agree. I'll try to come up with something else that is still simple. Don't know if I can though. I think the best use-case is when there's an `Update` monad involved.

Reply:
How about something like `filterA (\cid -> fmap (\c -> c.value == 5) (fetch cid))`. That probably doesn’t work as a doctest since it depends on external stuff but might at least give people an idea of where this could come in handy.

Reply:
Sure. I wanted to make the example in a way that it works. If that's not a requirement, something you suggested would totally work.

Reply:
I think the docs should primarily be useful for users. If we can test them automatically that’s great but we should opt for more useful docs over docs that might be automatically tested but are not useful to users. If you wan to test a full example you can throw it in the `daml-test-files` directory.